pkgs/applications/networking/instant-messengers/riot/riot-web.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, fetchpatch }:
+
+stdenv.mkDerivation rec {
+  name= "riot-web-${version}";
+  version = "0.12.2";
+
+  src = fetchurl {
+    url = "https://github.com/vector-im/riot-web/releases/download/v${version}/riot-v${version}.tar.gz";
+    sha256 = "0zyddpnng1vjli12hn1hd0w99g6sfsk80dn2ll5h9276nc677pnh";
+  };
+
+  installPhase = ''
+    mkdir -p $out/
+    cp -R . $out/
+  '';
+
+  meta = {
+    description = "A glossy Matrix collaboration client for the web";
+    homepage = http://riot.im/;
+    maintainers = with stdenv.lib.maintainers; [ bachp ];
```

Comment:
Since it is an io-only task, can you set `hydraPlatforms = [];` to skip building on hydra?

Reply:
Wouldn't that lead to having no binary package for it available? I think that is not desirable in case you have a machine that can only connect to the nixos cache but not to github directly.

Reply:
This is a common practice in nixpkgs we use for fonts and data only packages. I don't think it quite common to use matrix - a federated services - if you are not connected to the internet.

Reply:
I updated the PR